test/test_cli_integration.py

```diff
@@ -136,28 +143,31 @@ def generate_policy_string(self):
         variable_3 = 'simple/special @#$%^&*(){{}}[]._+/{id}'.format(id=uuid.uuid4().hex)
 
         policy = \
-"""
-- !variable
-  id: {variable_1}
-- !variable
-  id: {variable_2}
-- !variable
-  id: {variable_3}
-"""
+            """
```

Comment:
does this work? Usually """ has special formatting so please look at the generated policy file to make sure the indentation is correct

Reply:
yes, all tests passed
I dont think spaces are forbidden in a policy

Reply:
Not about all tests passing but how the actual policy is generated
```
            - !variable
              id: {variable_1}
            - !variable
              id: {variable_2}
            - !variable
              id: {variable_3}
            """
```
```
- !variable
id: {variable_1}
- !variable
id: {variable_2}
- !variable
id: {variable_3}
"""
```